lib/cli/generators/SVELTE/template/.storybook/config.js

```diff
@@ -0,0 +1,9 @@
+import { configure } from '@storybook/svelte';
+
+// automatically import all files ending in *.stories.js
+const req = require.context('../stories', true, /.stories.js$/);
```

Comment:
@Armanio I think there was a PR recently to turn `/.stories.js$/` into `/\.stories\.js$/` globally. Can you look for that and make the update here if it makes sense?

Reply:
Done.